shap/datasets.py

```diff
@@ -10,9 +10,24 @@
 github_data_url = "https://github.com/shap/shap/raw/master/data/"
 
 
-def imagenet50(display=False, resolution=224, n_points=None):
-    """ This is a set of 50 images representative of ImageNet images.
+def imagenet50(resolution=224, n_points=None):
+    """
+    This is a set of 50 images representative of ImageNet images.
+
+    Parameters:
+    - resolution (int, optional): The resolution of the images. Default is 224.
```

Comment:
Although this parameter has a default and therefore is an optional parameter, the _type_ of the variable is not Optional.  In the context of typing, [optional](https://docs.python.org/3/library/typing.html#typing.Optional) means that `None` is an accepted value. Here we have a regular `int`, not `int | None`.

It looks like `224` is the only resolution supported at present, so I suggest mentioning that in the parameter description.

Reply:
Sure, will correct this!